docs/HACKING.md

```diff
@@ -83,88 +90,67 @@ particular components.  This helps address reconfiguration of specific component
-
-The first time you enter the virtualenv, you should also install the project requirements, and some additional utilities.
+## Starting/Stopping Goldstone Server
 
-    $ pip install --upgrade pip
-    $ pip install -r requirements.txt
-    $ pip install -r test-requirements.txt
+To start the development environment, execute:
 
-If the requirements files change, you should rerun the `pip install` commands.
+    $ cd ~/devel/goldstone-server
+    $ ./bin/start_dev_env.sh
 
+The first time you start Golstone Server, it will probably take several minutes to download docker containers and perform configuraiton tasks.  You may see errors and missing data in the user interface. You may also see failures if you execute the test suite.  The data should be sufficiently populated in 10 minutes.  If you continue to see errors in the UI or in tests, please submit an issue!
 
-## Building the Goldstone Containers
+After the containers have started, you can access the user interface with the following information:
 
-All supporting services are available as docker containers. This step configures the Goldstone Server database, and is the final step before running the application.  It only needs to be done the first time you start Goldstone, and when you change ES or PostgreSQL schema.  You can rerun this step if you want to reinitialize, but realize that it will remove existing data in PostgreSQL and Elasticsearch.
+* url: http://127.0.0.1:8000
+* django admin username: admin
+* django admin password: goldstone
+* Goldstone admin username: gsadmin
+* Goldstone admin password: goldstone
 
-    $ cd $PROJECT_HOME/goldstone-server
-    $ bin/init_dev_env.sh
+To stop the development environment, either <Ctrl-C> from the window running `start_dev_env.sh` or run `stop_dev_env.sh` from another window. This will stop both the docker VM and the OpenStack VM.
 
-Here is an [example of the output](https://gist.github.com/jxstanford/8abace13d7ed2d036e09#file-init_dev_env-out) from a successful run.
-If you prefer to configure your own OpenStack, you will need to follow the instructions for configuring OpenStack hosts in the [INSTALL](http://goldstone-server.readthedocs.org/en/latest/INSTALL/) guide.  You should also update your `postactivate` script to use proper values for the `OS_*` settings.
 
-## Starting/Stopping Goldstone Server
+## Configuring OpenStack Instance
 
-There are some convenience scripts in `$PROJECT_HOME/goldstone-server/bin` for starting and stopping the virtual machines and docker containers that support the goldstone test environment.  For developer flexibilty, starting/stopping the django application has been omitted from the scripts.  To start the development environment, execute:
+With the development environment started in another window, execute the following command to configure the development OpenStack instance to ship logs and events back to Goldstone.  This only needs to be performed once:
```

Comment:
This doesn't agree with the text that follows it.  It says to "execute the following command," but then two commands are displayed.

It would help things if there was a concise name given to the start_dev_env terminal window, and another name given to the other window in which we run other scripts.  The instructions could say, "Do a start_dev_env in the xxxxx window, then switch to the eye window and do a configure_stack."  (Or fab runserver.)